Headers/AppKit/NSView.h

```diff
@@ -130,6 +130,7 @@ typedef enum _NSFocusRingType {
   id _boundsMatrix;
   id _matrixToWindow;
   id _matrixFromWindow;
+  id _coreAnimationBootstrapData;
```

Comment:
I think just `_coreAnimationData` is enough? What do you think?

Reply:
I am not sure about this. The code in NSView will need to decide when the execution should be transferred to the CA code. For this it needs more access to the internal structure. We could either define a proper interface for that or give it access to the data itself.
The current solution with just one id element won't work. I suggest to think about the interaction between gui and CA first and come up with a solution that covers this.